src/resolve/resolved-dns-server.h

```diff
@@ -13,8 +13,9 @@ typedef enum DnsServerType {
         DNS_SERVER_SYSTEM,
         DNS_SERVER_FALLBACK,
         DNS_SERVER_LINK,
+        _DNS_SERVER_TYPE_MAX,
```

Comment:
gcc seems to have started to complain that "enumeration value ‘_DNS_SERVER_TYPE_INVALID’ not handled in switch":
```sh
../src/resolve/resolved-dns-server.c: In function ‘dns_server_unlink’:
../src/resolve/resolved-dns-server.c:147:9: warning: enumeration value ‘_DNS_SERVER_TYPE_MAX’ not handled in switch [-Wswitch]
         switch (s->type) {
         ^~~~~~
../src/resolve/resolved-dns-server.c:147:9: warning: enumeration value ‘_DNS_SERVER_TYPE_INVALID’ not handled in switch [-Wswitch]
```

Reply:
@evverx Sorry. I have no idea why I missed this...
Fixed by #9706.

Reply:
Yeah, a warning in compilation with `-O0` would be pretty annoying → #9707.